README.md

```diff
@@ -3224,6 +3229,425 @@ port-channel hash input offset. Valid values are integer or 'default'.
 port-channel symmetry hash. Valid values are true, false or 'default'.
 
 --
+### Type: cisco_route_map
+
+Manages a Cisco Route Map.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(1)        | 1.6.0                  |
+| N3k      | 7.0(3)I2(1)        | 1.6.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.6.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.6.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.6.0                  |
+| N9k-F    | 7.0(3)F1(1)        | 1.6.0                  |
+
+#### <a name="cisco_interface-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:---------|:-------------|
+| `match_evpn_route_type_1`                | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_2_all`            | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_2_mac_ip`         | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_2_mac_only`       | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_3`                | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_4`                | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_5`                | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_6`                | Not supported on N3k,N9k-F,N9k  |
+| `match_evpn_route_type_all`              | Not supported on N3k,N9k-F,N9k  |
+| `match_length`                           | Not supported on N3k,N9k-F,N9k  |
+| `match_mac_list`                         | Not supported on N3k,N9k-F,N9k  |
+| `match_metric`                           | Not supported on N9k-F          |
+| `match_ospf_area`                        | Not supported on N3k,N5k,N6k,N9k-F,N9k <br> Supported in OS Version I5 and later on N3k,N9k  |
+| `match_vlan`                             | Not supported on N3k,N9k-F,N9k  |
+| `set_extcommunity_4bytes_additive`       | Not supported on N9k-F          |
+| `set_extcommunity_4bytes_non_transitive` | Not supported on N9k-F          |
+| `set_extcommunity_4bytes_transitive`     | Not supported on N9k-F          |
+| `set_extcommunity_cost_igp`              | Not supported on N9k-F          |
+| `set_extcommunity_cost_pre_bestpath`     | Not supported on N9k-F          |
+| `set_extcommunity_rt_additive`           | Not supported on N9k-F          |
+| `set_extcommunity_rt_asn`                | Not supported on N9k-F <br> Supported in OS Version I4 and before on N9k |

```

Comment:
Let's discuss each of these properties again that have limited OS version support.  I am fine with adding support for a specific version and later, but not a specific version and earlier.  We need to get fixes or eliminate support of this property all together.